docs/databricks-workflows.md

```diff
@@ -0,0 +1,135 @@
+# Running a dbt project as a job in Databricks Workflows
+
+Databricks Workflows is a highly-reliable, managed orchestrator that lets you author and schedule DAGs of notebooks, Python scripts as jobs. You can now schedule a dbt project to run as a production job.
+
+> The capability of running dbt in a Job is currently in private preview. You must be enrolled in the private preview to follow the steps in this guide. Features, capabilities and pricing may change at any time.
+
+In this guide, you will learn how to update an existing dbt project to run as a job, retrieve dbt run artifacts using an API and debug common issues.
+
+# Overview
+When you run a dbt project as a Databricks Job, the dbt Python process as well as the SQL generated by dbt run on the same Automated Cluster.
+
+If you want to run the SQL on, say, a Databricks SQL endpoint or even another cloud data warehouse, you can customize the checked-in `profiles.yml` file appropriately (see below).
+
+# Prerequisites
+- An existing dbt project version controlled in git
+- Access to a Databricks workspace
+- Ability to launch job clusters (using a policy or cluster create permissions) or access to an existing interactive cluster with `dbt-core` and `dbt-databricks` libraries installed or  `CAN_MANAGE` permissions to install the `dbt-core` and `dbt-databricks` as cluster libraries.
+- [Files in Repos](https://docs.databricks.com/repos/index.html#enable-support-for-arbitrary-files-in-databricks-repos) must be enabled and is only supported on Databricks Runtime (DBR) 8.4+ or DBR 11+ depending on the configuration. Please make sure the cluster has the appropriate DBR version.
+- Install and configure the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html)
+- Install [jq](https://stedolan.github.io/jq/download/), a popular open source tool for parsing JSON from the command line
```

Comment:
Would be good to mention that we recommend using DBR 10.4 or later versions for better sql compatibility.